commands/phantom.js

```diff
@@ -381,6 +381,16 @@ module.exports = {
     );
     return true;
   },
+  rejectTransaction: async () => {
+    const notificationPage = await playwright.switchToNotification(PROVIDER);
+    await playwright.waitAndClick(
+      PROVIDER,
+      transactionPageElements.buttons.rejectSign,
```

Comment:
[nit] would be nice to also have a new selector here :D or a more generic one:
Suggestions:
- buttons.primary
- buttons.rejectTransaction

Reply:
done!